SteamReview/SteamReview/Steam.cpp

```diff
@@ -0,0 +1,320 @@
+	}
+};
+
+//------------------------------------------------------
+void FSteam::OpenCategoryManager()
+{
+	std::system("cls");
+	std::cout << "-----------CATEGORY MANAGER-------------" << std::endl;
+	std::cout << "----------------------------------------" << std::endl;
+	std::cout << "1 - Create a new category" << std::endl;
+	std::cout << "2 - Delete a category" << std::endl;
+	std::cout << "3 - Main manu" << std::endl;
+	std::cout << "----------------------------------------" << std::endl;
+	std::cout << "choose an option" << std::endl;
+
+	int UserInput;
+	std::cin >> UserInput;
+	InputValidate(UserInput, 0, 4);
+
+	if (UserInput == 1)
+	{
+		if (AddNewCategory())
+		{
+			std::cout << "The category was added succesfully!" << std::endl;
+		}
+		else
+		{
+			std::cout << "something is wrong, the category wasn't be created..." << std::endl;
+		}
+
+		//back to MainMenu
+		std::cout << "Press enter to come back to Main menu" << std::endl;
+		if (std::cin.peek() == '\n')
+		{
+			ActiveMenuCommand = EMenuCommand::MainMenu;
+		}
+	}
+	else if (UserInput == 2)
+	{
+		if (DeleteACategory())
+		{
+			std::cout << "The category was deleted succesfully!" << std::endl;
+			
+		}
+		else
+		{
+			std::cout << "something is wrong, the category wasn't be deleted..." << std::endl;
+		}
+
+		//back to MainMenu
+		std::cin.ignore(500, '\n');
+		std::cout << "...Press enter to come back to Main menu..." << std::endl;
+		if (std::cin.peek() == '\n')
+		{
+			ActiveMenuCommand = EMenuCommand::MainMenu;
+		}
+	}
+	else
+	{
+		ActiveMenuCommand = EMenuCommand::MainMenu;
+	}
+};
+
+//------------------------------------------------------
+void FSteam::OpenDisplayGames()
+{
+	std::system("cls");
+	Uncategorized.DisplayCategory();
+	for (int i = 0; i < CategoryContainer.GetCurrentCategoryAmount(); i++)
+	{
+		std::cout << "----------------------------------------------------------------" << std::endl;
+		CategoryContainer.GetCategory(i).DisplayCategory();
+	}
+
+	std::cout << "----------------------------------------------------------------" << std::endl;
+	//back to MainMenu
+	std::cin.ignore(500, '\n');
+	std::cout << "Press enter to come back to Main menu" << std::endl;
+	if (std::cin.peek() == '\n')
+	{
+		ActiveMenuCommand = EMenuCommand::MainMenu;
+	}
+};
+
+//------------------------------------------------------
+void FSteam::AppRun()
+{
+	while (ActiveMenuCommand != EMenuCommand::Exit)
+	{
+		switch (ActiveMenuCommand)
+		{
+		case EMenuCommand::MainMenu:
+			OpenMainMenu();
+			break;
+		case EMenuCommand::AddGame:
+			OpenAddGame();
+			break;
+		case EMenuCommand::CategoryManager:
+			OpenCategoryManager();
+			break;
+		case EMenuCommand::DisplayGames:
+			OpenDisplayGames();
+			break;
+		case EMenuCommand::Exit:
+			break;
+		}
+	}
+}
+
+//---------------------------------------------------------------
+int FSteam::InputValidate(int InputValue, const int IntLowValueToValid, const int IntHighValueToValid)
```

Comment:
InputValidate is not a very clear function name at first glance. Remember that functions ideally start with a verb, so `ValidateInput` would be better.